pkgs/by-name/ve/versitygw/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "versitygw";
+  version = "1.0.8";
+
+  src = fetchFromGitHub {
+    owner = "versity";
+    repo = "versitygw";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-hrGPHl8vfLsL1JMcaU+RAQj6CaBJKsW+Q2AGCbgYUSA=";
+  };
+
+  vendorHash = "sha256-zZufxxZZ5Lfr0vWcygXdiRK+bhUr/+MFk4ajJGz5TMI=";
+
+  doCheck = false;
+
+  ldFlags = [
+    "-s"
+    "-w"
+  ];
+
```

Comment:
Would be better to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook if possible